src/config.ts

```diff
@@ -179,6 +181,8 @@ export const config = {
   retrySkipped: ms('1 minute'),
   retryBackoffPow: 3,
   refreshPeriod: ms('2 minutes'),
+  alternativeNamesNpmDownloadsThreshold: 5000,
+  alternativeNamesJsDelivrHitsThreshold: 10000,
```

Comment:
the jsDelivr threshold seems quite high, wouldn't this rule out eg. create-react-app, which only has 950 downloads on jsDelivr